### PR TITLE
Add 'PROJ4' package

### PR DESCRIPTION
**Pull request by michael-herwig**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/michael-herwig/hunter/build/1.0.2
  * https://travis-ci.org/michael-herwig/hunter/builds/375000441

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/240

### CODE REVIEW

File: docs/packages/pkg/PROJ4.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.. spelling::
+
+    PROJ4
+
+.. index:: GIS ; PROJ4
+
+.. _pkg.PROJ4:
+
+proj4
+===
```
Comment (ruslo) [inline docs/packages/pkg/PROJ4.rst]: `=====`

File: cmake/projects/PROJ4/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    PROJ4
+    VERSION
+    5.0.0
```
Comment (ruslo) [inline cmake/projects/PROJ4/hunter.cmake]: Should it be 5.0.1?

File: docs/packages/pkg/PROJ4.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+-  `Official <https://github.com/OSGeo/proj.4>`__
+-  `Hunterized <https://github.com/hunter-packages/PROJ4>`__
```
Comment (ruslo) [inline docs/packages/pkg/PROJ4.rst]: There is no hunterization fork.

File: cmake/projects/PROJ4/hunter.cmake (hunk 2 above)
Comment (michael-herwig) [reply cmake/projects/PROJ4/hunter.cmake]: It is kinda confusing but 5.0.0 seems to be right. Have a look [here](https://github.com/OSGeo/proj.4/issues/976)

File: docs/packages/pkg/PROJ4.rst (hunk 3 above)
Comment (michael-herwig) [reply docs/packages/pkg/PROJ4.rst]: How am i supposed to create one? Maybe i missed it in the tutorial:/ Or is this only for altered sources?

Comment (ruslo) [reply docs/packages/pkg/PROJ4.rst]: If you're using upstream sources there is no need to create a fork:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#cmake-no-dependencies

Simply remove this line from documentation.

**Comment by michael-herwig** [comment]
I fixed the issues, but I used --amned and push --force to retain convenience. However I cannot change the version to 5.0.1 because the project's cmake defines its version as 5.0.0 .

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3843
* https://travis-ci.org/ingenue/hunter/builds/375264639?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.20.67

**Pull request merged**